wpilibc/src/main/native/include/frc/smartdashboard/MechanismRoot2d.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <memory>
+
+#include <networktables/NetworkTableEntry.h>
+#include <wpi/Twine.h>
+
+#include "MechanismObject2d.h"
+
+namespace frc {
+
+/**
+ * Root Mechanism2d node.
+ *
+ * Do not create objects of this class directly! Obtain pointers from the
+ * Mechanism2d.GetRoot() factory method.
+ */
+class MechanismRoot2d : private MechanismObject2d {
+  friend class Mechanism2d;
+  struct private_init {};
+
+ public:
+  MechanismRoot2d(const wpi::Twine& name, double x, double y,
+                  const private_init& tag);
```

Comment:
You actually don't need to name the "tag" parameter, you can just leave it unnamed in both the declaration and definition.